types/modules.d.ts

```diff
@@ -23,3 +23,5 @@ declare module "abi-decoder" {
     export function decodeLogs(logs: any[]): DecodedLog[];
     export function decodeMethod(data: string): DecodedMethod;
 }
+
+declare module "sol-trace-set";
```

Comment:
the official way is supposed to be to create a folder name `sol-trace-set` in `types` and add this line to the `index.d.ts`

Reply:
same for `abi-decoder`